### ISSUE TITLE
REQ: Disable router setup/config via wireless client

### ISSUE THREAD

**Issue by RichardBu** [RMerl/asuswrt-merlin#408]
For security reasons I want to be able to disable setup of router via wireless, so you have to be connected via wired to change any settings.

**Comment by RichardBu** [comment]
Has this been closed because it has now been added, or because you don't plan on ever adding it ?

**Comment by RMerl** [comment]
I've closed all existing issues because after having kept it disabled for over a year due to the amount of noise, I recently decided to re-enable it, and rather than scan through those 2-4 years old issues, I've closed them all, and starting clean.

**Comment by RichardBu** [comment]
Should I create a new issue then with the same info and link back to this ?

**Comment by RMerl** [comment]
No, I'll re-open this one, so the back history can be kept.  Note that this still has a very low priority for me.  I did forward that request to Asus last year, in hopes that they might implement it themselves.  You currently can limit access by IP from the Administration -> System page, as a security feature.

**Comment by RichardBu** [comment]
OK, thanks for re-opening and forwarding request to Asus. I may have a go at contacting Asus myself as well.

**Comment by thezoggy** [comment]
why not disable the wifi setup ability if <insert some field/setting> has been set.. ? thus its enabled until its no longer needed.. seems like it would be a simple solution and security fix all in one?

**Comment by Zulgrib** [comment]
Can't we have a whitelist with mac adresses allowed to display the web gui ? Wouldn't that be easier ?

**Comment by RMerl** [comment]
There's already an IP whitelist on the webui, which has the benefit of working for both wired and wireless access control.  It's not the same as preventing access to any wireless client, which would be an added security level without the burden of managing your wired clients that want to allow through.

**Comment by azilber** [comment]
Here's a scenario which literally just happened to me.  I had to factory reset my Asus router because I was having issues with 380.64_2.  I switched to the beta firware, reset the router.   Apparently,  one of my neighbors has some device (most likely a phone) that automatically connects to SSID 'ASUS' almost immediately.  After rebooting my router 3x and trying to get in, I gave up and unscrewed the antennas from the router.  This allowed me to login.

Here's some options:
1) Make the SSID random.  That way, at least someone has to consciously connect to you, vs automatic trolling.  This may not be very useful though with WIFI+ on some phones being enabled.
2) Make the SSID hidden.  Just don't announce the default SSID.
3) Turn on WPS with some default code.

Literally, anything is better then the default open ASUS ssid.
